client/python/datajunction/client.py

```diff
@@ -304,6 +304,12 @@ def engines(self):
         response = self._session.get("/engines/", timeout=self._timeout)
         return response.json()
 
+    def metrics(self):
```

Comment:
Nice and scary. Wonder when we hit some perf issues when clients start calling this few times a second. Search APIs can surely come later...

Reply:
From what I can see, this largely depends on the backend database you're using. That being said, I have wondered if we can add some very light in-memory caching of this to at least cache intrasecond requests, but we have to think about the read-your-write behavior there and not make it confusing. I'll run some more performance tests soon and will think about this a bit more.